pyaptamer/experiments/_aptamer_eval.py

```diff
@@ -31,14 +117,14 @@ class Aptamer(BaseObject):
     --------
     >>> import torch
     >>> from pyaptamer.aptatrans import AptaTrans, EncoderPredictorConfig
-    >>> from pyaptamer.experiments import Aptamer
+    >>> from pyaptamer.experiments import AptamerEvalAptaTrans
     >>> apta_embedding = EncoderPredictorConfig(128, 16, max_len=128)
     >>> prot_embedding = EncoderPredictorConfig(128, 16, max_len=128)
     >>> model = AptaTrans(apta_embedding, prot_embedding)
     >>> target = "DHRNE"
     >>> device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
     >>> prot_words = {"AAA": 0.5, "AAC": 0.3, "AAG": 0.2}
```

Comment:
What `prot_words` is being used for is not clear in the docstring, also does the int values of the elements need to sum up to 1 or is it just a coincidence? If not i'd mention that.